css/scss/_fonts.scss

```diff
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: 'Roboto';
+  src: url(../fonts/Roboto-Regular.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Roboto';
+  src: url(../fonts/Roboto-Bold.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
```

Comment:
you need to pay attention to what git tells, as here ' no new line at the end of the file